x/proof/types/message_update_param_test.go

```diff
@@ -28,7 +28,7 @@ func TestMsgUpdateParam_ValidateBasic(t *testing.T) {
 			name: "invalid: param name incorrect (non-existent)",
 			msg: MsgUpdateParam{
 				Authority: sample.AccAddress(),
-				Name:      "WRONG_relay_difficulty_target_hash",
+				Name:      "WRONG_proof_missing_penalty",
```

Comment:
For consistency:

```suggestion
				Name:      "non_existent",
```

See [the updated params guide](https://github.com/pokt-network/poktroll/blob/5c995f12ea6957d57c7350f9186ff15bddcc971c/docusaurus/docs/develop/developer_guide/adding_params.md#52-parameter-update-tests):

![image](https://github.com/user-attachments/assets/bbf0744e-3e56-4da8-8afa-2ed8088ad324)